### DISCUSSION TITLE
Cannot use button after update primeng 17

### DISCUSSION THREAD

**Discussion by phuongpan** [primefaces/community#2679] [PrimeNG]
I recently updated angular 17 and PrimeNg 17 but when I try <p-button label="Primary" [rounded]="true" /> the button looks square seems like it does not have any primeng style

**Comment by qwadrox** [comment]
Hi. @phuongpan. Are you sure you're importing all styles correctly in angular.json?

```json
 "styles": [
    "src/styles.scss",
    "node_modules/primeicons/primeicons.css",
    "node_modules/primeng/resources/themes/lara-light-blue/theme.css",
    "node_modules/primeng/resources/primeng.min.css"
],
```
It has to work. 
I checked it for you in an example. Installed angular 17 with primeng 17: https://stackblitz.com/edit/6ag5gb

**Reply by phuongpan** [reply]
Hi @qwadrox, Thank you for your reply. I found the issue in the angular.json. Because I have a bootstrap, somehow, it overrides the primeng. I fix it by removing them from angular.json and separating them into layers in styles.scss.

`@import '~bootstrap/dist/css/bootstrap.css' layer(bootstrap);

@import "~primeng/resources/primeng.css" layer(primeng);`
